app/views/admin/cabinet_ministers/show.html.erb

```diff
@@ -1,6 +1,29 @@
 <% content_for :page_title, "Cabinet ministers ordering" %>
 <% content_for :title, "Cabinet ministers ordering" %>
 
+<% if reshuffle_in_progress? %>
+  <%= render "govuk_publishing_components/components/heading", {
+    text: "Preview ministers page in reshuffle mode",
+    heading_level: 2,
+    font_size: "m",
+    margin_bottom: 3,
+  } %>
+
+  <div class="govuk-!-margin-bottom-9">
+    <%= render "govuk_publishing_components/components/details", {
+      title: "Share ministers page preview",
+    } do %>
+      <% capture do %>
+        <%= render "govuk_publishing_components/components/copy_to_clipboard", {
+          label: "Copy and send this link to someone and they’ll be able to preview your draft on GOV.UK.",
+          copyable_content: "#{Plek.external_url_for('draft-origin')}/government/ministers",
+          button_text: "Copy link",
+        } %>
```

Comment:
I'm not 100% sure, on further reflection, that using the "shareable preview" styles with a non shareable link is a great idea. It might be a bit confusing for users who will expect it to work in the same way as shareable preview links. Would just a normal "Preview on website" link be better?